edx_sga/static/js/spec/main.js

```diff
@@ -0,0 +1,27 @@
+(function(requirejs, define) {
+    requirejs.config({
+        paths: {
+            'jquery': 'lib/jquery-2.1.4.js',
+            'sinon': 'js/lib/sinon-1.16.1',
+            'URI': 'js/lib/URI',
+            'underscore': 'js/lib/underscore'
```

Comment:
I don't think we'll be able to access those javascript files, the infrastructure in edx-platform isn't in place to do that cleanly. We may do this in a future issue but for now we need to bundle everything we need here